src/components/widget/input/cdvy-input.directive.js

```diff
@@ -120,7 +120,6 @@ class CodenvyInput {
       }
       // add ng-change attr if it exist in the parent element
       if ('ngChange' === key) {
-        inputElement.attr('data-ng-change', 'isChanged()');
```

Comment:
it seems that you previously added that block for a previous fix
also if you remove what is done inside the if () shouldn't we remove the whole if block ?